frontend/src/css/Generic.styl

```diff
@@ -1,6 +1,7 @@
 @import "../../node_modules/tracim_frontend_lib/src/css/Variable.styl"
 
 a
+  color black
```

Comment:
this commit https://github.com/tracim/tracim/commit/1bcc0c4d9f1997e01dccca7f8042d7d61e74d6e3#diff-603fe74d68ec0a8ae870de005c6e5cca actually did the exact opposite a few days ago.

The problem was to use the generic class .primaryColorFont on a link. It caused a specificity issue that made it impossible to have a link with the primaryColorFont.

So I removed the generic rule for links and we now have to set them manually.

You, then, have to remove this generic rule you just added and add some more specific rules to set the color of links that are still blue